Mathlib/Data/Nat/Log.lean

```diff
@@ -197,6 +197,29 @@ theorem add_pred_div_lt {b n : ℕ} (hb : 1 < b) (hn : 2 ≤ n) : (n + b - 1) /
     succ_pred_eq_of_pos (by omega)]
   exact Nat.add_le_mul hn hb
 
+lemma log2_eq_log_two {n : ℕ} : Nat.log2 n = Nat.log 2 n := by
+  induction n using Nat.strongRec
+  next n ih =>
```

Comment:
I suggest using ` induction ... with` here as well

Reply:
I attempted to the format you suggested but for whatever name I give after |, it seems to throw error "invalid alternative name 'n' " or such. Would it be possible to get a more detailed help?